service/pkg/server/services.go

```diff
@@ -179,10 +196,22 @@ func startServices(ctx context.Context, cfg config.Config, otdf *server.OpenTDFS
 	return nil
 }
 
+func tryExtractingServiceLoggerConfig(cfg serviceregistry.ServiceConfig) (string, error) {
```

Comment:
```suggestion
func extractServiceLoggerConfig(cfg serviceregistry.ServiceConfig) (string, error) {
```
<img width="528" alt="image" src="https://github.com/user-attachments/assets/ef3a8dde-87ef-481d-947a-769da9b81c32">